Example/Example/macOS/Code/NSStoryboard+Example.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  NSStoryboard+Example.swift
+//  Example
+//
+//  Created by Artem Starosvetskiy on 20/11/2016.
+//  Copyright © 2016 Artem Starosvetskiy. All rights reserved.
+//
+
+import Foundation
+import Cocoa
+
+extension NSStoryboard {
+
+    func instantiateController<ViewController>(_ type: ViewController.Type) -> ViewController {
+        return instantiateController(withIdentifier: String(describing: type)) as! ViewController
```

Comment:
I would not recommend use init(describing:). The result may depends on language version and other things.
As far as you inheriting ObjC classes, you can use NSStringFromClass, but it's totally up to you.